src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -4,7 +4,10 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Claims;
+using System.Security.Cryptography.Xml;
 using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Authentication;
```

Comment:
Not sure the changes in this file are needed

Reply:
Nope, just baggage from a previous implementation where it was done through an authentication handler.